rclcpp/src/rclcpp/parameter_service.cpp

```diff
@@ -33,6 +33,13 @@ ParameterService::ParameterService(
 {
   const std::string node_name = node_base->get_name();
 
+  rclcpp::IntraProcessSetting ipc_setting;
+  if (node_base->get_use_intra_process_default()) {
+    ipc_setting = rclcpp::IntraProcessSetting::Enable;
+  } else {
+    ipc_setting = rclcpp::IntraProcessSetting::Disable;
+  }
```

Comment:
do we need this block? woudln't the same logic be done in `create_service` where we call `resolve_intra_process` ?

Reply:
Removed in a436855